fudge-ratio/src/main/webapp/index.html

```diff
@@ -2,11 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>Fudge Ratio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <script src="script.js"></script>
+    <title>Fudge Ratio</title>
   </head>
-  <body>
-      <h1> Hello world!! Start files finished :-) </h1>
-      <p>Awesome!</p>
+  <body class="container">
+    <h1> Fudge Ratio Calculator </h1>
+    <p>Click the start button when you're ready!</p>
+    <br/><br/>
+    <button onclick="startTimer()">Start</button>
+    
+    <br/><br/>
```

Comment:
I also like to use $< br / >$, I wonder if we could do it with just JS (maybe if i have some free time I could do it)